app/homepage.py

```diff
@@ -57,6 +83,63 @@ def choose_map_style(n_clicks):
     return button_content_map, map_object, slider
 
 
+#This function returns the user selection area in the left side bar
+def user_selection_area():
+
+    return [build_alert_radio_button(),
+            dcc.Markdown('---'),
+            html.H5(("Filtres Carte"), style={'text-align': 'center'}),  # Map filters added here
+            html.P(build_layer_style_button()),                          # Changes layer view style btn
+            dcc.Markdown('---'),
+            html.P(build_map_style_button()),                            # Changes map style btn
+            html.P(id="hp_slider"),                                      # Opacity sliders for risks map
+            html.P(id="hp_alert_frame_metadata")                         # Displays alert_frame and alert_metadata
+            ]
+
+
+# Displays alert_frame and metadata related to a specific alert_markers after a click on display_alert_frame_btn
+def display_alerts_frames(feature=None):
+
+    # Fetching alert status and reusable metadata
+    alert_metadata = build_live_alerts_metadata()
+    alert_lat = str(alert_metadata["lat"])
```

Comment:
est-ce nécessaire de passer en str() les trois variables si tu les utilises dans une f-string ensuite ?

Reply:
Yep, good point. I was stucked trying to use `dcc.Markdown` that's why I tried to str them before giving up with this good old format 😄